content/root/best-practices-guide.textile

```diff
@@ -0,0 +1,97 @@
+---
+title: Best Practices Guide
+section: root
+index: 30
+jump_to:
+  Contents:
+    - Ably 101
+    - Using Ably like a pro
+---
+
+h2(#title). Ably Best Practices Guide
+
+In order to avail the best advantage of our platform, be sure to follow this best practices guide while building your applications using Ably.
+
+h3(#ably-101). Ably 101
+
+Ably Realtime is a cloud-based platform that allows you to easily integrate realtime functionality into your applications. Just plug and play.
+
+h4(#pubsub). Publish and Subscribe
+
+Publish data to a named "channel":/realtime/channels-messages on the Ably Realtime platform, and everyone subscribed to that same channel will receive it.
+
+h4(#notifications). Notify
+
+You can use "Push Notifications":/realtime/push to inform your users about the cool new features you are building in your application.
+
+h4(#presence). Be present
```

Comment:
Why not just "Presence"?  I appreciate this is meant to be 101,  but why is "Be present" (an action) simpler than "Presence" the feature and the same terminology we use everywhere else in our documentation.